azure-pipelines.yml

```diff
@@ -0,0 +1,17 @@
+# Starter pipeline
```

Comment:
Hi ei2kpi, it looks like this is specific to your pipeline, not the mrtk pipeline. Could you please remove it? @MenelvagorMilsom please correct me if I'm wrong here.

Reply:
Yeah, this isn't part of the mrtk pipelines and can not run on our builds. Possibly an unintentional addition?